rust-plugins/react/Cargo.toml

```diff
@@ -11,8 +11,8 @@ farmfe_core = { version = "*", path = "../../crates/core" }
 farmfe_macro_plugin = { version = "*", path = "../../crates/macro_plugin" }
 farmfe_toolkit_plugin_types = { version = "*", path = "../../crates/toolkit_plugin_types" }
 serde = { version = "1.0", features = ["derive"] }
+farmfe_toolkit = { path = "../../crates/toolkit" }
```

Comment:
plugin should not relies on `farmfe_toolkit` to avoid swc status conflicts, use `farmfe_core` instead